config.example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "token": "discord.bot.token.here",
+  "database_uri": "mongodb://",
+  "admins": [ "your user id here" ],
+  "prefix": "c!",
+  "mainGuild": "your guild id here",
+  "color": 12404274,
+  "webhookUrl": null,
+  "port": 1001
```

Comment:
Don't document this, it's only for the live version

Reply:
Done.